### PR TITLE
Address Ondrei's and Lorenzo's comments

### PR DESCRIPTION
**Pull request by furry13**
- "Start CLAT" section:
- - when to start
- IPv6 Address section:
- - it should be a separate address
- - the address shall be checksum-neutral and random
- - DAD shall be used
- IPv4 address section:
- - clarifying wired/wireless terminology
- - require 192.0.0.0/29 usage for the latter
- - one address per CLAT

.... and some other changes

### CODE REVIEW

File: draft-link-v6ops-464xlaton.xml
Comment (mstojens) [inline draft-link-v6ops-464xlaton.xml:235]:
```suggestion
It means that it might not be possible to run more than 8 CLAT instances per node.
```

And might means definitely not, right? I can't see a way to have more than eight with the existing requirements.

Comment (mstojens) [inline draft-link-v6ops-464xlaton.xml:235]: 8, or 6? Are we reusing the first and last address from this range as just another source address? See https://www.rfc-editor.org/rfc/rfc6333#section-10

File: draft-link-v6ops-464xlaton.xml
Comment (mstojens) [inline draft-link-v6ops-464xlaton.xml:228]: There's also nothing stopping the "wireless network architecture" mode being used with wired networks, right? the "wireless" is a misnomer tied to the 3GPP original bias?

File: draft-link-v6ops-464xlaton.xml
Comment (mstojens) [inline draft-link-v6ops-464xlaton.xml:219]:
```suggestion
Wireline network architecture, as defined in Section 4.1 of <xref target="RFC6877"/>. In that case the node performing CLAT functions also extends the network downstream and provides network connectivity services to other connected systems.
```

File: draft-link-v6ops-464xlaton.xml
Comment (mstojens) [inline draft-link-v6ops-464xlaton.xml:226]:
```suggestion
Wireless network architecture, as defined in Section 4.2 of <xref target="RFC6877"/>.
```

File: draft-link-v6ops-464xlaton.xml
Comment (furry13) [reply draft-link-v6ops-464xlaton.xml]: In case of CLAT those addresses  are internal to the host and never appears on the wire, so .0, .1 and .7 can be used. MacOS, for example, configures that address with 255.255.255.255 mask:
inet 192.0.0.2 netmask 0xffffffff broadcast 192.0.0.2
As per https://www.rfc-editor.org/rfc/rfc7335.html#section-4

"A host MUST NOT
   enable two active IPv4 continuity solutions simultaneously in a way
   that would cause a node to have overlapping 192.0.0.0/29 address
   space."

Therefore .1 can't be used as DS-lite router if 464xlat is present.

Let me add some text (in another PR) about that.

File: draft-link-v6ops-464xlaton.xml
Comment (furry13) [reply draft-link-v6ops-464xlaton.xml:228]: Good point, I didn't make it clear - kind of implied by "desktops" but it's not enough!
Updated the text. What do you think?

Comment (mstojens) [reply draft-link-v6ops-464xlaton.xml:228]: Love it, thanks!

File: draft-link-v6ops-464xlaton.xml
Comment (furry13) [reply draft-link-v6ops-464xlaton.xml]: fixed!

**Comment by furry13** [comment]
BTW let me know if/when you think it's ready to be merged - I'll do it and submit -01 to the Datatracker.

**Comment by mstojens** [comment]
I think it's 99% there, I need to suggest text for one more line that outdated I think with the edits since, but I'll have that posted in the next 12 hours, I promise. Otherwise, it LGTM. Thanks for taking all my chaotic feedback!

Also, do you want me to do a separate PR for grammatical/spelling polish tomorrow after you merge, or is that too much work for the -01? Your call.

**Comment by furry13** [comment]
> I think it's 99% there, I need to suggest text for one more line that outdated I think with the edits since, but I'll have that posted in the next 12 hours, I promise. Otherwise, it LGTM. Thanks for taking all my chaotic feedback!

Thanks for reviewing!

>Also, do you want me to do a separate PR for grammatical/spelling polish tomorrow after you merge, or is that too much work for the -01? Your call.

Pls send me a PR for fixing my "Runglish" ;) Thanks!

**Comment by mstojens** [comment]
Ok, so either you fixed whatever I saw in December that I wanted to revisit, or I made it up because after a thorough re-read, I'm not finding the inconsistency I thought I saw earlier. Let's merge!

**Review by mstojens — APPROVED** [review]

**Comment by furry13** [comment]
> Ok, so either you fixed whatever I saw in December that I wanted to revisit, or I made it up because after a thorough re-read, I'm not finding the inconsistency I thought I saw earlier. Let's merge!

Oh I know what you are talking about...It was unfinished sentence, wasn't it? - I did fix it - I thought I responded with a comment but it looks like I didn't ;( Sorry for the confusion.

>I'll do a final edit pass after you merge so I can fork off the main branch.

Awesome, thank you!

**Pull request merged**
